src/server/req_runjob.c

```diff
@@ -1970,6 +1970,17 @@ req_defschedreply(struct batch_request *preq)
 	reply_send(preq);
 }
 
+/**
+ * @brief
+ *	convert_job_to_resv - creates a reservation out of the job
+ * 			      and moved the job to the newly created
```

Comment:
I think you want the present tense here (move).  Also, I you might want to make creates not plural.